.github/issue_template/bug.md

```diff
@@ -0,0 +1,56 @@
+---
+name: 🐛 Bug Report
+about: Submit a bug report to help us improve
+labels: "bug"
+---
+
+<!--
+Note: if you are reporting a wrong signature of a function or a class in
+the standard library, then the typeshed tracker is better suited
+for this report: https://github.com/python/typeshed/issues
+-->
+
+## 🐛 Bug Report
+
+(A clear and concise description of what the bug is.)
+
+### Have you read the [Contributing Guidelines](https://github.com/python/mypy/blob/master/CONTRIBUTING.md)?
```

Comment:
Why should people who report a bug read the contributing guidelines?

Reply:
@JelleZijlstra I have removed the contributing guidelines link from all the issue templates.